src/platform/forms/save-in-progress/actions.js

```diff
@@ -142,7 +142,7 @@ export function migrateFormData(savedData, migrations) {
   let savedVersion = savedData.metadata.version;
   while (typeof migrations[savedVersion] === 'function') {
     savedDataCopy = migrations[savedVersion](savedDataCopy);
-    savedVersion++; // eslint-disable-line no-param-reassign
```

Comment:
Just a bit of cleanup while I was looking at it; this is no longer a param, so no need to ignore it. :man_shrugging: